README.md

```diff
@@ -1,34 +1,166 @@
-* General Specs: Place at the main level of your directory. Try to follow the same file name standards as others.
-* Multiple components: If you have multiple components, your main component should be your general repo name (i.e. byu-news-components) and then inside should be a folder for each component (i.e. byu-news and byu-story).
-* Component Specific Specs: Place component-specific specs in it's own component folder for good organization.
-Please try to be as detailed with your specs as possible. They need to be complete to have good products.
+    * The specs should specify how the design handles common and unique cases.
+
+
+# Site Wiring
+
+Whenever the `master` branch is pushed to Github, Github pages will run Jekyll on our site source and produce HTML that will be served by Github Pages.
+
+Jekyll requires a specific [site layout](https://jekyllrb.com/docs/structure/):
+
+* `_config.yml` - contains Jekyll's configuration settings, including the top-level navigation.
+* `Gemfile` - Used to run Jekyll locally.
+* `index.md` - Placeholder for the site homepage. Content is actually in `_layouts/home.html`.
+* `_includes/` - Blocks that are available to be included in other pages. See [the docs](https://jekyllrb.com/docs/includes/) for more.
+* `_layouts/` - Base layouts for the rendered pages
+    * `default.html` - Basic layout
+    * `home.html` - the actual HTML for the site's homepage
+    * `spec-home.html` - Layout for the home page of a specification category
+    * `spec.html` - Layout for a specification page.
+* `_sass/` - SASS files. These are included for inclusion in the top-level SCSS files (in `assets`), and are not directly rendered to CSS.
+    * `theme/` - SCSS related to the main page layout and theme.
+        * `_colors.scss` - Common colors
+        * `_layout.scss` - top-level layout
+        * `_mixins.scss` - common mixins
+        * `_type.scss` - Typography
+    * `blocks/` - styles for the helper blocks (see above).
+    *  `site.scss` - pulls together everything that goes in the site-wide CSS
+* `_site/` - where the rendered site files go when running Jekyll locally
+* `_specs_*/` - Links to the directories under `specs/`. Makes Jekyll work while maintaining in a nice organization of our code.
+* `assets/` - site-wide assets. Text assets here, including SCSS files, must start with [YAML Front Matter](https://jekyllrb.com/docs/frontmatter/).
+* `readme-images/` - images that are used in the `README.md` file, which is not included in the site.
+
+# Running Jekyll Locally
+
+## Setup
+
+You must have [Ruby](https://www.ruby-lang.org/en/) and [Bundler](http://bundler.io/) installed.
+
+### macOS
+
+Ruby is already installed.
+
+```
+gem install bundler
+```
+
+## Running
+
+```
+bundle exec jekyll serve --incremental
+```
+
+If you don't want to remember that, and are on Linux or macOS, just run `run.sh`.
```

Comment:
When I run `run.sh` with or without `sudo` I get the error `sh: 0: Can't open run`. I assume it's user error, but I wanted to share this experience just in case.

Reply:
Could you do an `ls -l run.sh` and post the output?

Reply:
`-rwxrwxr-x 1 btm296 btm296 178 Jan 19 14:26 run.sh`